lib/color_spinor_util.cu

```diff
@@ -424,4 +424,29 @@ namespace quda {
     }
   }
 
+  void resize(std::vector<ColorSpinorField> &v, size_t new_size, const ColorSpinorParam &param)
```

Comment:
not related to this change but wondering if color_cpinor_util could become a cpp file

Reply:
That is my plan.  It's not possible at the moment because a variety of the headers included here expect a device aware compiler.  E.g., `math_helper.cuh` doesn't work if included in gcc.  I intend to fix this....